src/Runners/PHPUnit/Configuration.php

```diff
@@ -134,7 +134,7 @@ public function getSuiteByName(string $suiteName)
                                     unset($excludedPaths[$dir]);
                                 }
                             }
-                            // not breaking on purpose
+                            // no break on purpose
```

Comment:
This appeases phpcs-fixer as it can only be configured with exactly one acceptable phrase for this particular rule.